### PR TITLE
WifiNetworkSuggestion API for android 10 and above.

### PR DESCRIPTION
**Pull request by sauravkumal**
Implements wifi network suggestion from this documentation:
[https://developer.android.com/develop/connectivity/wifi/wifi-suggest](https://developer.android.com/develop/connectivity/wifi/wifi-suggest)

- Tested on android 14

Suggestions are welcome :)

### CODE REVIEW

**Comment by sauravkumal** [comment]
@gustavoabel updated

**Comment by gustavoabel** [comment]
@sauravkumal Please fix the linting errors so we can proceed with the PR.

File: lib/types/index.d.ts [hunk 1]
```diff
@@ -8,6 +8,13 @@ declare module 'react-native-wifi-reborn' {
         timestamp: number;
     };
 
+    export type SuggestedNetworkConfig = {
+        ssid: string;
+        password?: string;
+        isWpa3?: boolean;
+        isAppInteractionRequired?: boolean;
+    }
```
Comment (gustavoabel) [inline lib/types/index.d.ts:16]: modify here please:
```suggestion
    };
```

**Review by gustavoabel — APPROVED** [review]

**Pull request merged**